msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md

```diff
@@ -282,6 +282,98 @@ The following table includes the response codes:
 | **403** | The app is unable to send the signal. 403 response code can occur because of various reasons, such as the tenant admin disables and blocks the app during live site migration. In this case, the payload contains a detailed error message. |
 | **404** | The meeting chat doesn't exist. |
 
+## Targeted meeting notification API
+
+The `targetedMeetingNotification` API allows apps to send notifications to specific participants in a meeting. Apps send targeted meeting notifications based on user action. The API is available through bot API.
+
+> [!NOTE]
+>
+> * The API payload only permits a task module with a URL.
+> * The user ID formats **aadObjectid** and **UPN** aren't supported.
+
+Get supported user ID format for targeted meeting notification:
+
+* [Get participant API](#get-participant-api)
+* [Get members API](../bots/how-to/get-teams-context.md#fetch-the-roster-or-user-profile)
+
+### Example
+
+Following is an example of request payload:
+
+```http
+GET POST /v1/meetings/{meetingId}/notification
+```
+
+```json
+{
+
+  "type": "targetedMeetingNotification",
+  "value": {
+    "recipients": [ 
+"29:1I12M_iy2wTa97T6LbjTh4rJCWrtw2PZ3lxpD3yFv8j2YPnweY2lpCPPAn3RI0PP7rghfHauUz48I1t7ANhj4CA"
+     ], 
+    "surfaces": [ 
+      { 
+        "surface": "meetingStage", 
+        "contentType": "task", 
+        "content": { 
+          "value": { 
+            "height": "300", 
+            "width": "400", 
+            "title": "Targeted meeting Notification", 
+            "url": "https://somevalidurl.com"           
+}
+        } 
+      } 
+    ] 
+  },
+  "channelData": { // optional if a developer wants to support user attributes
```

Comment:
Shouldn't this be // optional if a developer does not want to support user attributes?

Reply:
updated as "optional if a developer doesn't want to support user attributes".